src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/BlockCollection.js

```diff
@@ -457,7 +519,10 @@ class BlockCollection<T: string, U: {type: T}> extends React.Component<Props<T,
                     className={blockCollectionStyles.addButton}
                     disabled={disabled || this.hasMaximumReached}
                     icon="su-plus"
-                    onClick={this.handleAddBlock}
+                    // eslint-disable-next-line react/jsx-no-bind
+                    onClick={(value) => {
+                        this.handleAddBlock(value);
+                    }}
```

Comment:
we should instead define the method this binding correctly instead of eslint ignore here something